modules/color.py

```diff
@@ -14,6 +14,9 @@ async def color(interaction: discord.Interaction, color: str = None) -> None:
       await interaction.followup.send("Role color set to default")
       return
     
+    if color in COLORS:
```

Comment:
this needs to handle lower case colors.

also since `COLORS` is a dictionary, we can just use `COLORS.get(color.lower(), color)` here and it would be more efficient